src/Control/Abstract/Modules.hs

```diff
@@ -116,8 +116,8 @@ instance ( Has (Reader (ModuleTable (Module (ModuleResult address value)))) sig
     case op of
       Load    name  k -> askModuleTable >>= maybeM (throwLoadError (ModuleNotFoundError name)) . fmap moduleBody . ModuleTable.lookup name >>= k
       Lookup  path  k -> askModuleTable >>= k . fmap moduleBody . ModuleTable.lookup path
-      Resolve names k -> k (find (`Set.member` paths) names)
-      List    dir   k -> k (filter ((dir ==) . takeDirectory) (toList paths))
+      Resolve names k -> k (find (`Set.member` paths) (map Path.absRel names))
+      List    dir   k -> k (filter ((Path.absRel dir ==) . Path.takeDirectory) (toList paths))
```

Comment:
I'd also prefer to avoid `absRel` calls here, if possible.

Reply:
Sorry, I did not notice this comment.  I will refactor it this afternoon.

Reply:
Refactored by comparing with String.  Future refactor will go with next PR with Module definition.

Reply:
Oops, I forgot to push the commit.  The related commit is in dc90cc950.